docs/static/releasenotes.asciidoc

```diff
@@ -53,17 +53,128 @@ This section summarizes the changes in the following releases:
 
 [[logstash-7-15-0]]
 === Logstash 7.15.0 Release Notes
-coming[7.15.0]
 
 [[featured-7-15-0]]
 ==== New features and enhancements
 
+[[ecs-7-15-0]]
+===== Progress toward Elastic Common Schema (ECS)
+
+In this release, we continued our efforts towards Elastic Common Schema (ECS):
+
+* <<plugins-codecs-collectd,collectd codec>> added `target` support
+* <<plugins-codecs-edn,edn codec>> added `target` support
+* <<plugins-codecs-edn_lines,edn_lines codec>> added `target` support
+* <<plugins-codecs-fluent,fluent codec>> added `target` support
+* <<plugins-codecs-json,json codec>> added `target` support and `ecs_compatibility`
+* <<plugins-codecs-json_lines,json_lines codec>> added `target` support and `ecs_compatibility`
+* <<plugins-codecs-line,line codec>> added `target` support and `ecs_compatibility`
+* <<plugins-codecs-msgpack,msgpack codec>> added `target` support
+* <<plugins-codecs-multiline,multiline codec>> added `target` support and `ecs_compatibility`
+* <<plugins-codecs-plain,plain codec>> added `target` support and `ecs_compatibility`
+* <<plugins-filters-translate,translate filter>> supports `ecs_compatibility`
+* <<plugins-inputs-heartbeat,heartbeat input>> supports `ecs_compatibility`
+
+Check out our https://github.com/elastic/logstash/issues/11635[progress toward ECS compatibility] in github issue https://github.com/elastic/logstash/issues/11635[#11635].
+
 [[notable-7-15-0]]
 ==== Performance improvements and notable issues fixed
 
+* Updated Bundler to latest version https://github.com/elastic/logstash/pull/13141[#13141]
```

Comment:
@robbavey there's more indicated for "Release notes" in https://github.com/elastic/logstash/pull/13015
wasn't sure the (internal) locking would provide much value for users -> but maybe it just needs a better story

Reply:
It's a good question - while this should not have any material on impact on users, it might be worth a note for developers, explaining that we can now keep up to date with latest bundler versions, and calling out the new `--enable-local-plugin-development` setting.

Reply:
The flag is mentioned just one line bellow this one:
```markdown
* There's a new developer-only flag (`--enable-local-plugin-development`) to facilitate local plugin development
```

> might be worth a note for developers, explaining that we can now keep up to date with latest bundler versions

Yeah so I somehow wanted to expand on having latest and greatest Bundler but it always came back a bit technical,
```suggestion
* Updated Bundler to latest version https://github.com/elastic/logstash/pull/13141[#13141]
  - Bundler 2.2.26 brings along improvements and bug fixes for Logstash's dependency resolution
  - We plan to continue being up-to-date with latest Bundler and/or RubyGems in the future
```
would this work or any suggestions?

Reply:
🤦 And missing lines like this is why I should always look at the full file, and not just the edited highlights from github...

And yeah, I like the suggestion, that works